mkosi/__init__.py

```diff
@@ -1910,11 +1928,22 @@ def build_uki(
         )
 
         if sign_expected_pcr:
-            cmd += [
-                "--pcr-private-key", context.config.secure_boot_key,
-                "--pcr-banks", "sha1,sha256",
-            ]
-            options += ["--ro-bind", context.config.secure_boot_key, context.config.secure_boot_key]
+            cmd += ["--pcr-banks", "sha1,sha256"]
```

Comment:
Didn't we just change swtpm to only have sha256 banks, could this be an issue?

Reply:
dunno, but the banks are not changed by this PR as you can see in the diff